lib/smart_proxy_pulp_plugin/pulpcore_plugin.rb

```diff
@@ -6,21 +6,24 @@ class PulpcorePlugin < ::Proxy::Plugin
     default_settings :pulp_url => 'https://localhost',
                      :content_app_url => 'https://localhost:24816/',
                      :mirror => false,
-                     :client_authentication => ['password', 'client_certificate']
+                     :client_authentication => ['password', 'client_certificate'],
+                     :rhsm_url => 'https://localhost/rhsm'
 
     AUTH_TYPES = ['password', 'client_certificate', 'client_certificate_admin_only'].freeze
 
     load_validators include: ::PulpProxy::Validators::Include
     validate :pulp_url, :url => true
     validate :content_app_url, :url => true
     validate :client_authentication, :include => AUTH_TYPES
+    validate :rhsm_url, :url => true
 
     expose_setting :pulp_url
     expose_setting :mirror
     expose_setting :content_app_url
     expose_setting :username
     expose_setting :password
     expose_setting :client_authentication
+    expose_setting :rhsm_base_url
```

Comment:
I think the `_base` is breaking you here as `rhsm_url` is used elsewhere.

Reply:
Yup, that was it

```
12 tests, 33 assertions, 0 failures, 0 errors, 0 pendings, 0 omissions, 0 notifications
100% passed
```

TY